tests/test_cli.py

```diff
@@ -119,3 +119,48 @@ def test_cli_wrong_definitions_name():
         ],
     )
     assert result_valid.exit_code == 1
+
+
+def test_cli_custom_dimensions():
+    """Check that CLI runs through with a non-default dimension"""
+
+    result_valid = runner.invoke(
+        cli,
+        [
+            "validate-project",
+            str(TEST_DATA_DIR / "non-default_dimensions_passing"),
+            "--dimensions",
+            "['variable', 'region', 'foo']",
```

Comment:
Using a folder called `"foo"` is fine in principle just one thing to note about the functionality of `nomenclature` in general.
The dimension that is validated is determined by the name of the **folder**, this means that in your case even though the folder `foo` contains a file called `scenarios.yaml` this would be used to validate a, usually non-existent, column called `foo`. 
For the tests it's fine to keep it this way though.

Reply:
I think you raise a good point @phackstock - let us use names for "passing" tests that are similar to what is expected by users (ie, IAMC columns, region, variable, scenario) and use "foo" for failing tests, non-existing folders, etc.

Reply:
Good to know thank you